bestpy/best.py

```diff
@@ -1,14 +1,16 @@
-from .answers import answers
-
 import random
+from collections import abc
+
+from .answers import answers
 
 
 class Best:
     def __getattribute__(self, name):
         answer = answers[name]
-        if callable(answer):
-            answer = answer()
 
-        if isinstance(answer, list):
+        if callable(answer):
+            return answer()
```

Comment:
```suggestion
            answer = answer()
```
This doesn't look quite right. We're updating the previous variable to get the return value and continue the execution with it.

Reply:
Yep, not quite sure why I made that change.